rest/admin_api.go

```diff
@@ -445,34 +445,178 @@ func (h *handler) handlePutDbConfig() (err error) {
 // GET database config sync function
 func (h *handler) handleGetDbConfigSync() error {
 	h.assertAdminOnly()
-	// TODO: STUB
-	h.writeJSONStatus(http.StatusOK, `function(doc, oldDoc) {
-	// this is a stub sync function
-}`)
+	var (
```

Comment:
Do we also need to support DELETE for the sync and import_filter endpoints?

Reply:
Sure. I still need to find a way of reducing code duplication between all of these endpoints modifying a single field, but copy-pasted for now to get them working for the beta.